src/Clock.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+type Props = {
+  clockName: string
+};
+
+type State = {
+  nowDate: Date;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    nowDate: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const newValue = new Date();
```

Comment:
Do you really need to write a new Date into this constant? It is better to write it directly in the state, isn't it?
`this.setState({ nowDate: new Date() });`